ember-core/src/main/scala/org/http4s/ember/core/Parser.scala

```diff
@@ -432,6 +444,30 @@ private[ember] object Parser {
         (EmptyBody.covary[F], (Some(buffer): Option[Array[Byte]]).pure[F]).pure[F]
       }
 
+    def parseUnknownBody[F[_]: Concurrent](
```

Comment:
Should this be marked private, to avoid it being binary-published?

Reply:
I'm not sure, what is the concern there? Happy to change it to whatever is preferred, I was just taking cues from `parseFixedBody`.

Reply:
>  But it doesn't appear to be used anywhere outside

It isn't used outside `Parser`, but it is defined in `Body` and used in `Response`. It seems appropriate to define in `Body`, so we could make it `private[Parser]` but that doesn't seem to generate the same unused compiler warnings as `private` does.

Reply:
:+1: Let's just leave it alone then.